JWPlayer-Developer-Demo/SwiftPlayerViewController.swift

```diff
@@ -77,68 +79,71 @@ class SwiftPlayerViewController: UIViewController, JWPlayerDelegate {
         
         //MARK: JWCaptionStyling
         let captionStyling: JWCaptionStyling = JWCaptionStyling()
-        captionStyling.font = UIFont (name: "Zapfino", size: 20)
-        captionStyling.edgeStyle = raised
-        captionStyling.windowColor = UIColor.purple
+        captionStyling.font            = UIFont (name: "Zapfino", size: 20)
+        captionStyling.edgeStyle       = JWEdgeStyleRaised
+        captionStyling.windowColor     = .purple
         captionStyling.backgroundColor = UIColor(red: 0.3, green: 0.6, blue: 0.3, alpha: 0.7)
-        captionStyling.fontColor = UIColor.blue
-        config.captionStyling = captionStyling
+        captionStyling.color           = .blue
+        config.captions                = captionStyling
         
         //MARK: JWAdConfig
         let adConfig: JWAdConfig = JWAdConfig()
-        adConfig.adMessage = "Ad duration countdown xx"
+        adConfig.adMessage   = "Ad duration countdown xx"
         adConfig.skipMessage = "Skip in xx"
-        adConfig.skipText = "Move on"
-        adConfig.skipOffset = 3
-        adConfig.adClient = vastPlugin
-        config.adConfig = adConfig
+        adConfig.skipText    = "Move on"
+        adConfig.skipOffset  = 3
+        adConfig.client      = JWAdClientVast
+        
         
         /* auto start */
         //        config.autostart = true
         
         //MARK: Waterfall Tags
         let waterfallTags: NSArray = ["bad tag", "another bad tag", "http://playertest.longtailvideo.com/adtags/preroll_newer.xml"]
-        
+
         //MARK: Ad Schedule
-        config.adSchedule = [JWAdBreak(tags:waterfallTags as! [String], offset:"1"),
+        adConfig.schedule = [JWAdBreak(tags:waterfallTags as! [String], offset:"1"),
                              JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"5"),
-                             JWAdBreak(tag: "http://demo.jwplayer.com/player-demos/assets/overlay.xml", offset: "7", nonLinear: true),
-                             //                           JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"0:00:05"),
-            JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"50%"),
-            JWAdBreak(tag: "http://playertest.longtailvideo.com/adtags/preroll_newer.xml", offset:"post")]
+//                             JWAdBreak(tag: "http://demo.jwplayer.com/player-demos/assets/overlay.xml", offset: "7", nonLinear: true),
```

Comment:
if we won't be using this, might as well delete it

Reply:
@AmitaiB I don't like spaces as well. As for including the SDK in the demo, maybe it's better to use CocoaPods or something like that?